packages/api-openfin/notification.html

```diff
@@ -11,13 +11,20 @@
       right: 0px;
       bottom: 0px;
     }
+
+    h4, p {
+      padding: 0px;
+      margin: 0px;
```

Comment:
Even more pedantic... `0px` is consistent with the styles above, which is good, but the `px` is unnecessary; something to consider removing. I.e. `padding: 0;` etc. is sufficient and a little less cluttered IMO.